api/sql/schema/750-rule.decision.fetch.sql

```diff
@@ -154,6 +157,7 @@ BEGIN TRY
         )
     ORDER BY
         c.[priority],
+        c.[name],
```

Comment:
The priority is not required to be unique, so the ordering is also by name, for the conditions with the same priority.